Nurse Tasks/Preview Content/data.json

```diff
@@ -0,0 +1,132 @@
+[{
+    "clientName": "client1",
+    "assignedTo":"Nurse1",
+    "location": {
+        "street":"ABC St",
+        "city":"Burnaby"
+    },
+    "startTime":{
+        "year":"2023",
+        "month":"03",
+        "day":"01",
+        "hour":"15",
+        "mintue":"00"
+    },
+    "endTime":{
+        "year":"2023",
+        "month":"04",
+        "day":"01",
+        "hour":"15",
+        "mintue":"00"
+    },
+    "taskTitle": "task1",
+    "notes":"",
+    "type":"type1",
```

Comment:
For the types, can you make some of them 'work' and some 'personal'? That will help me with my ticket :)